### ISSUE TITLE
Missing optional argument in console.profileEnd

### ISSUE THREAD

**Issue by krzysztof-grzybek** [Microsoft/TypeScript#24869]
**TypeScript Version:**  3.0.0-dev.20180609

**Search Terms:** profileEnd console missing argument profile reportName

**Code**

```ts
console.profileEnd('myReportName')
```
https://github.com/Microsoft/TypeScript/blob/ba4bf21eade396ac0d82ab1278dfddc6ce69c653/lib/lib.webworker.d.ts#L323

**Expected behavior:** Type checker shouldn't complain about argument, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Console/profileEnd)

**Actual behavior:** Type checker complains about argument. There is missing optional argument for `console.profileEnd` definition.

**Playground Link:**  

**Related Issues:**
